front/app/containers/Admin/reporting/components/ReportBuilder/Editor/index.tsx

```diff
@@ -1,7 +1,8 @@
 import React from 'react';
 
 import { Box } from '@citizenlab/cl2-component-library';
-import { Editor as CraftEditor } from '@craftjs/core';
+import { Editor as CraftEditor, QueryMethods } from '@craftjs/core';
```

Comment:
It required adding `@craftjs/core` to `package.json` to fix `dependency-cruiser` error.